crypto/fipsmodule/ml_kem/ml_kem_ref/kem.c

```diff
@@ -34,6 +49,11 @@ int crypto_kem_keypair_derand(ml_kem_params *params,
   hash_h(sk+params->secret_key_bytes-2*KYBER_SYMBYTES, pk, params->public_key_bytes);
   /* Value z for pseudo-random output on reject */
   memcpy(sk+params->secret_key_bytes-KYBER_SYMBYTES, coins+KYBER_SYMBYTES, KYBER_SYMBYTES);
+
+  // Abort in case of PCT failure.
+  if (keygen_pct(params, pk, sk)) {
+    BORINGSSL_FIPS_abort();
```

Comment:
* The declaration of `BORINGSSL_FIPS_abort` is [several directories up](https://github.com/aws/aws-lc/blob/90d2a34ead8d51c3a97be33f1e6eff751707807c/crypto/internal.h#L1270):
```
#include "./../../../internal.h"
```
* `BORINGSSL_FIPS_abort` is only available in a FIPS build, but the logic should abort regardless.
```
  if (keygen_pct(params, pk, sk)) {
#if defined(BORINGSSL_FIPS)
    BORINGSSL_FIPS_abort();
#else
    abort();
#endif
  }
```
*

Reply:
NP: It would be more clear if the header files from this source directory (at the top of the file) more clearly indicated that they're relative paths:
```
#include "./params.h"
#include "./kem.h"
#include "./indcpa.h"
#include "./verify.h"
#include "./reduce.h"
#include "./symmetric.h"
```